runestone/parsons/parsons.py

```diff
@@ -25,9 +25,9 @@ def setup(app):
     app.add_directive('parsonsprob', ParsonsProblem)
     app.add_node(ParsonsNode, html=(visit_parsons_node, depart_parsons_node))
     app.add_stylesheet('parsons.css')
-    app.add_stylesheet('lib/prettify.css')
-    app.add_javascript('lib/prettify.js')
-    app.add_javascript('lib/hammer.min.js')
+    app.add_stylesheet('js_lib/prettify.css')
+    app.add_javascript('js_lib/prettify.js')
+    app.add_javascript('js_lib/hammer.min.js')
```

Comment:
What does this fix?

Reply:
Oddly, when I look at the files in `_static/` produced by Sphinx/runestone build, there is no `lib/` subdirectory and the .js files that should be in `lib/` aren't present. The rename means that the `js_lib/` subdirectory and all the expected files are present. This is all when using Sphinx 1.7.4. It's so odd that using `lib/` doesn't work, but it just doesn't. Can you verify this behavior on Linux/Mac?

Reply:
Oops, I'm (partly) wrong on this. On a branch I'll (eventually) merge, this bug appears and this PR fixes it; however, it doesn't appear when not using my branch. I'll do more investigation.

Reply:
OK, I figured this out. I had a `lib/` directory containing library files to exclude from the build, but this also excluded all JS in the `lib/` directory of the Parson's problems. Ouch.

To avoid namespace/directory naming problems like this, I'd suggest keeping this change (use the name `js_lib/`) to make it a more distinct name. @bnmnetp, what's your preference?

Reply:
Is there any reason that those files are in a lib folder?  Why can't they all just live directly in the js folder?

Reply:
Good point. I'm not the author of the Parson's problems, but FWIW: the `lib/` subdirectory includes files not written by Runestone people, but copied from other projects. Hence, the idea of putting these files in a place to indicate they're a dependency makes sense...but perhaps using another name.